README.md

```diff
@@ -14,23 +15,21 @@ Repository to describe, develop, document and test the DeviceStatus API family
   * check if a device is losing connection to the network or gets reachable again, and the roaming status.
   * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
 * Describe, develop, document and test the APIs (with 1-2 Telcos)  
-* Started: July 2022
-* Location: virtually  
+* Started: July 2022 
 
-## Meetings
-* Meetings are held virtually
-* Schedule: bi-weekly (odd weeks), Wednesday, 11 AM CET
-* Meeting link: <a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_NDQwYTlkZjYtOTc0NS00MjA5LWE2YjItMWFjYTczZTM4NTEx%40thread.v2/0?context=%7b%22Tid%22%3a%22bde4dffc-4b60-4cf6-8b04-a5eeb25f5c4f%22%2c%22Oid%22%3a%22389ab01b-6fbd-4e02-8f1a-1be1e09c95e8%22%7d">Microsoft Teams Meeting</a>
-
-## Status and released versions
-* Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
-
-* The latest available and released version 0.5.1 of the API is available within the [release-0.5.1 branch](https://github.com/camaraproject/DeviceStatus/tree/release-0.5.1)
+## Contributing
+* Meetings
+  * Bi-weekly on Wednesday, 11:00 CET/CEST (08:00 UTC, 09:00 UTC during European DST)
+  * [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94783050047?password=c43ff9fd-4c79-468a-9d98-45222dd6343d)
+* Maling List
+  * Subscribe <sp-dst+subscribe@lists.camaraproject.org>
+  * <https://lists.camaraproject.org/g/sp-dst>
+* Lifecycle Status
+  * [Latest Release](https://github.com/camaraproject/DeviceStatus/releases/latest)
+  * Release Information: Link to wiki
+   * Note: Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until a new release is created. For example, changes may be reverted before a release is created. **For best results, use the latest available release**.
+   * The latest available and released version 0.5.1 of the API is available within the [release-0.5.1 branch](https://github.com/camaraproject/DeviceStatus/tree/release-0.5.1)
   - API definition with inline documentation:
     - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceStatus/blob/release-0.5.1/code/API_definitions/device-status.yaml)
     - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/release-0.5.1/code/API_definitions/device-status.yaml&nocors)
     - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceStatus/release-0.5.1/code/API_definitions/device-status.yaml) 
```

Comment:
These lines we might need to be only in CHANGELOG.md and the release notes in Github (to avoid that the README need to be updated with each release).

Reply:
Kept in for now. Added comment in https://github.com/camaraproject/DeviceStatus/pull/139/commits/16e59f107bc13d9fbe199a53369dee16aae1d6dd that the lines can be omitted going forward.